Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/BaseInputHandler.cs

```diff
@@ -14,6 +14,8 @@ public abstract class BaseInputHandler : InputSystemGlobalHandlerListener
         [Tooltip("Is Focus required to receive input events on this GameObject?")]
         private bool isFocusRequired = true;
 
+        private bool activeFocusValue = true;
```

Comment:
For clarity, I'd call this `areHandlersRegistered`

Reply:
Also, you should probably check this value in `OnDisable()` instead of checking `isFocusRequired`.

Reply:
Good catch.